src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/4156db
-spring.datasource.username=
-spring.datasource.password=
+spring.datasource.username=root
```

Comment:
idk why it's here after it's been added to git ignore

Reply:
Try https://www.christianengvall.se/untrack-files-local-repository-keep-origin-git/#:~:text=To%20untrack%20files%20in%20local,in%20the%20repository%20when%20cloning.

Reply:
worked:)